### PR TITLE
feat(it-tools): add it-tools container

### PR DESCRIPTION
**Pull request by polarstack**
**Description**
Add it-tools container https://github.com/CorentinTh/it-tools to create a chart for it

⚒️ Fixes  # 

**⚙️ Type of change**

- [x] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [x] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: mirror/it-tools/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+LABEL org.opencontainers.image.source=https://github.com/truecharts/containers
+ARG CONTAINER_NAME
+ARG CONTAINER_VER
+LABEL org.opencontainers.image.licenses="BSD-3-Clause"
+LABEL org.opencontainers.image.title="${CONTAINER_NAME}"
+LABEL org.opencontainers.image.url="https://truecharts.org/docs/charts/${CONTAINER_NAME}"
+LABEL org.opencontainers.image.version="${CONTAINER_VER}"
+LABEL org.opencontainers.image.description="Container for ${CONTAINER_NAME} by TrueCharts"
+LABEL org.opencontainers.image.authors="TrueCharts"
+LABEL org.opencontainers.image.documentation="https://truecharts.org/docs/charts/${CONTAINER_NAME}"
```
Comment (stavros-k) [inline mirror/it-tools/Dockerfile:11]:
```suggestion
LABEL org.opencontainers.image.documentation="https://truecharts.org/docs/charts/${CONTAINER_NAME}"

```

File: mirror/it-tools/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM corentinth/it-tools:2023.5.14-77f2efc@sha256:9038f48a679b8396ce2cf5bf783822228229a77fa3ce20b4fc3b815558551e03
```
Comment (stavros-k) [inline mirror/it-tools/Dockerfile:1]: With that kind of tag... a regex versioning would be needed.
Check renovate.json5 file for examples.

Comment (stavros-k) [reply mirror/it-tools/Dockerfile:1]: Nope.

Comment (polarstack) [reply mirror/it-tools/Dockerfile:1]: So something like this?

```
// matchPackagePatterns
    {
      "matchDatasources": ["docker"],
      "versioning": "regex:^v(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)-(?<digest>\\d+)$",
      "matchPackageNames": ["corentinth/it-tools"],
      "automerge": true,
    },
```

**v2023.5.14-77f2efc**

v=hardcoded by package maintainer
2023=major
5=minor
14=patch
77f2efc=digest

Comment (stavros-k) [reply mirror/it-tools/Dockerfile:1]: You can replace the digest with `.+`
It's not needed, the regex is only for "sorting" so renovate can pick the "newer" release.
The digest will be done using the sha256 of the actual image

Comment (stavros-k) [reply mirror/it-tools/Dockerfile:1]: `(?<.+>\\d+)` -> `.+`

Comment (polarstack) [reply mirror/it-tools/Dockerfile:1]: got it, thx a lot. commit to renovate.json5 done

**Review by stavros-k — APPROVED** [review]

**Pull request merged**